lib/schema_generator/format.rb

```diff
@@ -290,11 +291,10 @@ def required_links
 
       def frontend_properties
         links.each_with_object({}) do |(k, v), hash|
-          # It's possible for all link types to contain items without base_paths
-          # however apps aren't coded for this so fail on this, therefore
+          # It's possible for all link types to contain items without base_paths# however apps aren't coded for this so fail on this, therefore
```

Comment:
Something happened here with two lines of comments...

Reply:
Gah. Thanks!  Got to massage this through a bunch of temperamental e2e tests again now 😬